audio/libshout/Portfile

```diff
@@ -38,6 +37,15 @@ patchfiles          apple-patches \
 # Fix implicit declaration of functions and fix build with ccache.
 use_autoreconf      yes
 
-configure.args      --enable-shared \
-                    --enable-static \
-                    --with-pic
+configure.args-append \
+                --with-pic \
+                --enable-shared \
+                --enable-static \
+                --program-suffix=1 \
+                --libdir=${prefix}/lib/shout1 \
+                --includedir=${prefix}/include/shout1 \
+                --x-includes=${prefix}/include \
+                --x-libraries=${prefix}/lib
+
+# This is an old version, so disable livecheck
+livecheck.type  none
```

Comment:
Ideally the indentation of the entire portfile should be consistent. The rest of the lines are indented to 20 spaces but your new lines are indented to 16 spaces. Could the new lines be changed to 20 spaces to match?

Often we like to sort configure args alphabetically. The existing args were sorted. Can we sort the new args?

You added `--program-suffix=1` but I don't think this is necessary because this port does not install any programs.

You added `--x-includes=${prefix}/include` and `--x-libraries=${prefix}/lib` but I don't think this is necessary because this port does not depend on any X11 libraries.

```suggestion
configure.args-append \
                    --enable-shared \
                    --enable-static \
                    --includedir=${prefix}/include/shout1 \
                    --libdir=${prefix}/lib/shout1 \
                    --with-pic

# This is an old version, so disable livecheck
livecheck.type      none
```